src/workloads/query/ArrayTraversal.yml

```diff
@@ -227,14 +227,6 @@ AutoRun:
   - When:
       mongodb_setup:
         $eq:
-          - standalone
-          - standalone-80-feature-flags
-          - standalone-all-feature-flags
-          - standalone-classic-query-engine
-          - standalone-heuristic-bonsai
-          - standalone-query-stats
-          - standalone-sampling-bonsai
-          - standalone-sbe
```

Comment:
`standalone-sbe` and `standalone-classic-query-engine` are still in use, we might not want to delete those (or check with query)